### DISCUSSION TITLE
Help test Netzero app redesign

### DISCUSSION THREAD

**Discussion by zigam** [netzero-labs/netzero#40] [Announcements]
Hi!

We're getting ready to roll out a big redesign of the Netzero app, improving app navigation and look & feel. If you're interested in early access to the iOS app, send us an email at info@netzero.energy and include your Apple ID email address in the message.

You'll get a [TestFlight](https://developer.apple.com/testflight/) invitation which will allow you to run beta versions of the app and provide feedback.

This does not affect the Netzero backend, so your automations will continue to run. You'll also have access to the old [web interface](https://app.netzero.energy/), and ability to downgrade the installed version to the currently live version if you run into issues.

Android and web redesign will follow soon.

Thanks!
Ziga

**Comment by BJReplay** [comment]
Hanging out for Android when you get to it.

Love this app!

**Reply by eJonny** [reply]
What are the goals for the redesign? The responsiveness (when scrolling through different time periods) is significantly better! The new UI is clean, but maybe not the most efficient use of screen space (for example the "live section" of the dashboard is wasted space when looking at prior periods). The new UI seems to be more intuitive and discoverable.

**Reply by zigam** [reply]
Thanks for the feedback!

The main goal for the redesign was improved navigation, but we also wanted to polish the app a bit and improve the look & feel. The current style with top navigation drop-downs is non-standard, doesn't scale well, and is not very discoverable. The new app has bottom and top navigation clearly labeled and one tap away. The Settings screen can also scale better now.

A lot of work also went into the Automation UI, which is probably the most complex part of the UI (lots of components and screens). The new UI opens up space for more automations with clearer descriptions.

Agreed about the live section wasted space, we'll fix that. One of the downsides of new navigation is also that the charts are smaller; we're planning on improving that with a "full screen" view, especially useful for always-on iPad dashboards.

**Reply by zigam** [reply]
@BJReplay you can now join the Beta if you visit the Netzero app page in Play Store:
https://play.google.com/store/apps/details?id=com.zigam.pwmon

I don't see the new release yet, it might take Play Store some time to update.

**Reply by BJReplay** [reply]
Option to join beta available and accepted.

**Reply by BJReplay** [reply]
And the beta is now available and downloading.

Will review and report back.

How would you like us to report feedback and issues?

One issue per report / suggestion?

Or one combined report after using it for a few days?

**Reply by BJReplay** [reply]
I've got to say that it looks great.  One niggle - will raise an issue.

**Reply by zigam** [reply]
Doesn't matter how feedback is reported, whatever is easiest for you, thanks!
